### ISSUE TITLE
Starting Up React Create-React-App my-app doesnt work with react-router

### ISSUE THREAD

**Issue by leonin72** [omniscientjs/omniscient#135]
As soon as I try to use any of the tags of react router eg LINKS or The Routes I can only see the HTML which is blank. I tried the simplest used. The single route. That doesn't even work.

**Comment by mikaelbr** [comment]
Could you post your code so we can reproduce it? We haven't used react-router much, and especially not the latest version, so it isn't developed with compatibility with RR explicitly in mind.

@torgeir You have tried omniscient with an earlier version of RR, right?

**Comment by torgeir** [comment]
Yes, I believe we have some old examples laying around, but haven't tried with more recent versions.

Please post an example so we can reproduce it.
